pyproject.toml

```diff
@@ -22,6 +22,10 @@ classifiers = [
 python = ">=3.8, <3.11"
 aiohttp = "^3.8.1"
 msgpack = "^1.0.3"
+grpcio = "^1.48.1"
+grpcio-tools = "^1.48.1"
+protobuf = "^3.20.1"
```

Comment:
I don't think protobuf needs to be added as a separate dependency. It comes with `grpcio-tools`

Reply:
i asked her to add it to freeze the version of protobuf.  we can remove it. problem is, if we install protobuf manually, then it installs an incompatible version with grpcio-tools.

Reply:
But, we don't need to install protobuf manually right?

Reply:
let's remove it.